src/app/organizer/ItemTable.tsx

```diff
@@ -628,48 +635,32 @@ function ItemTable({
         }
       : undefined;
 
+  const onMoveSelectedItems = (store) => {
+    const items = selectedFlatRows?.map((d) => d.original);
+    for (const item of items) {
+      moveItemTo(item, store, false, item.amount);
+    }
+  };
+
+  const onTagSelectedItems = (tagInfo: TagInfo) => {
+    const items = selectedFlatRows?.map((d) => d.original);
```

Comment:
This can end up undefined - maybe check and skip doing anything if selectedFlatRows is undefined?